devbook.xsl

```diff
@@ -710,7 +700,7 @@
                 <xsl:with-param name="append">../</xsl:with-param>
               </xsl:call-template>
             </xsl:variable>
-            <a href="{concat($relative_path_depth_recursion, $relative_path, 'index.html')}"><xsl:value-of select="document(concat($parentItem_actual, 'text.xml'))/guide/chapter[1]/title"/> &#160;<span class="fa fa-arrow-right"/></a>
+            <a class="w-200 text-center" href="{concat($relative_path_depth_recursion, $relative_path, 'index.html')}"> <span class="truncated-text d-inline-block max-w-150 mr-1"><xsl:value-of select="document(concat($parentItem_actual, 'text.xml'))/guide/chapter[1]/title"/></span><span class="fa fa-arrow-right"/></a>
```

Comment:
```suggestion
            <a class="w-200 text-center" href="{concat($relative_path_depth_recursion, $relative_path, 'index.html')}"> <span class="truncated-text d-inline-block max-w-150 mr-1"><xsl:value-of select="document(concat($parentItem_actual, 'text.xml'))/guide/chapter[1]/title"/></span>&#160;<span class="fa fa-arrow-right"/></a>
```
This looks somewhat cramped, so I'd suggest to leave the space in place.
Same for `fa-arrow-left` below (three times).

Reply:
I have replaced ml-1 and mr-1 by ml-2 and mr-2. This way it should look pretty much as before now.